docs/releasing.md

```diff
@@ -39,7 +39,8 @@ We may introduce forward-compatible changes: query parameters and response prope
 
 ## Releasing a version
 
-1. Check out the appropriate release branch `x.y`.
+1. Update the translations from Transifex, see [README](../locales/README.md#pulling-translation-updates-from-transifex).
+2. Check out the appropriate release branch `x.y`.
 2. Make sure `CHANGELOG.md` is up to date.
```

Comment:
Do you want to propagate the numbering increments through the rest of this list?

Reply:
oops. though in reality github doesn't care about misordered lists IIRC and will render them fine. better not to though:)